website/client-old/js/services/guideServices.js

```diff
@@ -64,11 +64,15 @@ function($rootScope, User, $timeout, $state, Analytics) {
     classes: [
       [
         {
+          orphan: true,
+          content: window.env.t('classGearText'),
+          final: true,
           state: 'options.inventory.equipment',
           element: '.equipment-tab',
           title: window.env.t('classGear'),
-          content: window.env.t('classGearText')
-        }, {
+          hideNavigation: true
+        }
+        /*, {
```

Comment:
@crookedneighbor Do you know if any of the following tour elements are still used in the code?